platform/windows/context_gl_windows.cpp

```diff
@@ -33,6 +33,9 @@
 // Author: Juan Linietsky <reduzio@gmail.com>, (C) 2008
 
 #include "context_gl_windows.h"
+#include <dwmapi.h>
+
+#pragma comment(lib, "dwmapi.lib")
```

Comment:
What does this do? If you want to link `dwmapi.lib`, it should be added to the `LIBS` array in `platform/windows/detect.py`.